src/auto/initable.rs

```diff
@@ -0,0 +1,50 @@
+// This file was generated by gir (5a68ad0) from gir-files (469db10)
+// DO NOT EDIT
+
+use Cancellable;
+use Error;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct Initable(Object<ffi::GInitable, ffi::GInitableIface>);
+
+    match fn {
+        get_type => || ffi::g_initable_get_type(),
+    }
+}
+
+impl Initable {
```

Comment:
Unusable :(

Reply:
Yes this needs special support, also the async one. It's a different way of creating objects that implement this interface (constructing them can fail).

Reply:
Meaning, someone need to think about a design how to expose GInitiable and GAsyncInitiable in Rust generally :)